lib/registrant.rb

```diff
@@ -0,0 +1,24 @@
+#Create a Registrant class that has "name, age, permit, and license_data" attributes 
+#so our vistors can use services.
+
+class Registrant
+  attr_accessor :name, 
+                :age, 
+                :permit, 
+                :license_data
+
+  def initialize(name, age, permit = false)
+  @age = age
+  @name = name
+  @permit = permit
+  @license_data = {:written=>false, :license=>false, :renewed=>false}
+  end
+
+  def permit?
+    @permit
+  end
+
+  def earn_permit
+    @permit = true
+  end
+end
```

Comment:
I don't believe any of these attributes need to be listed as attr_accessors because we don't need to change their values outside of this class. Also, I don't think you need any type of reader or accessor for `permit` since you never need to call `.permit` on a REgistrant object. 

Make sure to indent lines 11-14